### PR TITLE
WIP: pycall

### PR DESCRIPTION
**Pull request by AlexRobson**

### CODE REVIEW

File: src/SimulationData.jl [hunk 1]
```diff
@@ -4,7 +4,15 @@ using DataDeps
...
 
+export
+    read_estimate,
+    read_distribution,
+    read_array,
+    read_table
+
+
 include("units.jl")
+include("pyapi.jl")
```
Comment (sdl1) [inline src/SimulationData.jl:15]: Hey, I think this file is missing from the PR?

Comment (AlexRobson) [reply src/SimulationData.jl:15]: should be included now

Comment (AlexRobson) [reply src/SimulationData.jl:15]: I don't get why it's not appearing in the files changed. It's in the file tree:

https://github.com/ScottishCovidResponse/SimulationData.jl/blob/ar/pycall/src/pyapi.jl

Comment (sdl1) [reply src/SimulationData.jl:15]: Probably related to the outage today, maybe make a new PR and close this one?

**Comment by AlexRobson** [comment]
Closing in favour of https://github.com/ScottishCovidResponse/SimulationData.jl/pull/2

**Pull request closed without merging**